KarhooUISDK/Screens/PassengerDetails/PassengerDetailsViewController.swift

```diff
@@ -313,13 +329,13 @@ extension PassengerDetailsViewController: KarhooInputViewDelegate {
     private func getValidInputViewCount() -> Int {
         var validSet = Set<String>()
         
-        for (_, inputView) in inputViews.enumerated() {
+        inputViews.forEach({ inputView in
             if inputView.isValid() {
                 validSet.insert(inputView.accessibilityIdentifier!)
```

Comment:
Force unwrapping in not necessary here. I'd add default value ("") just to limit potential production crash cases.